test/e2e/specs/editor-tracking/editor-tracking__block-inserted-event.ts

```diff
@@ -283,12 +283,43 @@ describe(
 					await fullSiteEditorPage.selectExistingTemplatePartFromModal( 'header-centered' );
 				} );
 
-				it( '"wpcom_block_instered" event does NOT fire', async function () {
+				// The wp_block_inserted event does fire here because the
+				// header block selected above includes a core/page-list
+				// block, which triggers wpcom_block_inserted. This is
+				// arguably a reasonable outcome. We need to decide whether
+				// to adjust the test to match the tracking behavior or adjust
+				// the underlyting tracking behavior.
+				it.skip( '"wpcom_block_instered" event does NOT fire', async function () {
 					const eventDidFire = await editorTracksEventManager.didEventFire(
 						'wpcom_block_inserted'
 					);
 					expect( eventDidFire ).toBe( false );
 				} );
+
+				it( 'Add a Page List block to the header template part', async function () {
+					const openInlineInserter: OpenInlineInserter = async () => {
+						await headerBlock.clickAddBlockButton();
+					};
+					await fullSiteEditorPage.addBlockInline(
+						'Page List',
+						'[aria-label="Block: Page List"]',
+						openInlineInserter
+					);
+				} );
+
+				it( '"wpcom_block_inserted" event fires with correct "entity_context" and "template_part_id"', async function () {
+					const eventDidFire = await editorTracksEventManager.didEventFire(
+						'wpcom_block_inserted',
+						{
+							matchingProperties: {
+								block_name: 'core/page-list',
+								entity_context: 'core/template-part/header',
+								template_part_id: 'pub/blockbase//header-centered',
+							},
+						}
+					);
+					expect( eventDidFire ).toBe( true );
+				} );
```

Comment:
It looks like we are adding new tests here as well?

The first seems like it is just setup for the second? And it doesn't expect/assert anything. I wonder if it makes more sense to combine those into one test or actually `expect` something in that first one?

Reply:
We had added these tests at a point where we were going to be skipping other tests in this file that tested inserted blocks within general template parts. Those tests were not working due to issues with entity_context. Header template parts did not have the same issues, so I recreated the tests here. 

We've since resolved the underlying entity_context issue and reenabled skipped tests for general template parts. I had been thinking to keep these new tests as they're working and just add more coverage. But after discussion, we've decided it would be better to avoid adding redundant tests that we'll need to maintain going forward. 

So I'll remove these now. Thanks!